modular-docs-manual/content/topics/module_modular-documentation-repositories.adoc

```diff
@@ -0,0 +1,141 @@
+[id="modular-documentation-repositories"]
+= Modular Documentation Repositories
+
+IMPORTANT: As CCS strategic imperatives and Pantheon 2 development continue to evolve, the details and recommendations in this section and possibly other sections in this guide might be modified. The Flexible Customer Content (FCC) team recognizes that the fluctuation in recommendations can be frustrating for CCS teams trying to adopt modularization standards and apologizes in advance as the initiative pursues the best possible outcome for CCS documentation.
+
+Due to many factors that determine documentation repository design for individual Red Hat products, the Flexible Customer Content (FCC) initiative cannot provide a strict documentation repository template for all CCS teams to follow. Each team faces different implementation challenges, especially when Red Hat product documentation is single-sourced with corresponding community project documentation.
+
+However, for optimal FCC, Pantheon, and Pantheon 2 compliance, the FCC initiative recommends a general repository structure similar to the following example, where possible:
+
+.Recommended documentation repository structure (where possible)
+[source]
+----
+├── _artifacts
+├── _images
+├── assemblies
+    ├── _artifacts -> ../_artifacts  // Symlinks can be extracted.
+    ├── _images -> ../_images
+    ├── modules -> ../modules
+    ├── assembly-kogito-configuring.adoc
+    ├── assembly-kogito-creating-running.adoc
+    ├── assembly-kogito-decision-engine.adoc
+    ├── assembly-kogito-developing-decision-services.adoc
+    ├── assembly-kogito-developing-process-services.adoc
+    ├── assembly-kogito-using-dmn.adoc
+    ├── assembly-kogito-using-drl.adoc
+    ├── ...
+├── modules
+    ├── configuration
+        ├── con-data-index-service.adoc
+        ├── con-jobs-service.adoc
+        ├── proc-data-index-service-enable-security.adoc
+        ├── ref-data-index-service-queries.adoc
+        ├── ...
+    ├── creating-running
+        ├── con-kogito-automation.adoc
+        ├── con-kogito-modelers.adoc
+        ├── proc-kogito-app-examples-running.adoc
+        ├── ref-kogito-app-examples.adoc
+        ├── ...
+    ├── decision-services  // Contains modules related to multiple assemblies.
+        ├── con-decision-tables.adoc
+        ├── con-dmn.adoc
+        ├── con-drl.adoc
+        ├── proc-dmn-model-creating.adoc
+        ├── ref-dmn-model-example.adoc
+        ├── ...
+    ├── process-services
+        ├── ...
+    ├── ...
+├── titles-enterprise
+    ├── kogito-configuring
+        ├── _images -> ../../_images
+        ├── assemblies -> ../../assemblies
+        ├── master.adoc
+        └── master-docinfo.xml
+    ├── kogito-creating-running
+        ├── ...
+    ├── kogito-developing-decision-services  // Contains multiple assemblies.
+        ├── ...
+    ├── ...
+├── pantheon2.yml
+└── README.md
+----
+
+This repository structure contains the following key components:
```

Comment:
I'd suggest using callouts for these descriptions.

Reply:
Haha, funny you say that. There has been explicit guidance from the style team to avoid callouts where possible, for accessibility reasons and for minimizing clutter. So I'm afraid I'm stuck between two conflicting opinions on this.